score.py

```diff
@@ -128,6 +134,109 @@ def run_inference(images_with_header: List[ImageWithHeader],
     return segmentation
 
 
+def extract_zipped_dicom_series(zip_file_path: Path, extraction_folder: Path) -> None:
+    """
+    Unzip a zip file assumed to contain a single DICOM series.
+
+    All files are extracted directly into the extraction folder discarding any folders they
+    may have in the zip file.
+
+    :param zip_file_path: Path to zip file.
+    :param extraction_folder: Path to extraction folder.
+    """
+    with zipfile.ZipFile(zip_file_path, 'r') as zip_file:
+        for zipped_file in zip_file.infolist():
+            if not zipped_file.is_dir():
+                # discard the path, if any, to just get the filename and suffix
+                zipped_file.filename = os.path.basename(zipped_file.filename)
+                zip_file.extract(zipped_file, str(extraction_folder))
```

Comment:
What if file names clash? If they do, how do the error messages look like? Are they actionable? (ie., are they telling the user what needs to be fixed?)